### PR TITLE
Retool outreach email

### PR DESCRIPTION
**Pull request by bpf120**
First go. I need a follow up

### CODE REVIEW

File: 01-04-2023-version-control-your-retool-app.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ICP: "VP ENG, CTO, Devs"
+Market: "Any company using Retool"
+
+---
+
+Hi First Name,
+
+Retool is a powerful tool that allows low-code solution to create internal tools. I'm sure you are aware as a user :)
+
+What if you added a version controlled database to that process? You no longer have to worry about accidently making changes to your production database. Maybe today you work off of a copy or a backup. But how do you manage changes to your database?
```
Comment (timsehn) [inline 01-04-2023-version-control-your-retool-app.md]:
```suggestion
What if you added a version controlled database to your Retool applications? You no longer have to worry about accidentally making changes to your production database. Maybe today you work off of a copy or a backup. But how do you manage changes to your database?
```

File: 01-04-2023-version-control-your-retool-app.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Dolt allows you to do just that. Commits, branches, and pull requests are built into Dolt. If you use it as a backing store for our Retool app, then you will get those features in your app automatically.
```
Comment (timsehn) [inline 01-04-2023-version-control-your-retool-app.md]:
```suggestion
Dolt allows you to do just that. Commits, branches, and pull requests are built into Dolt. If you use Dolt as the database for your Retool app, then you will get those features in your app automatically. 
```

**Review by timsehn — COMMENTED** [review]
Made some edit suggestions.

How you gonna find Retool custoemrs?

**Comment by bpf120** [comment]
Thx. I'll use their website and stackshare to find customers.

**Pull request merged**